Sources/UILabelExtensions.swift

```diff
@@ -65,6 +65,19 @@ extension UILabel {
             self.text = _text
         }, completion: nil)
     }
+   
+    // Set lineSpacing for UILabel
+    public func setLineSpacing(lineSpacing: CGFloat) {
+        let paragraphStyle = NSMutableParagraphStyle()
+        paragraphStyle.lineSpacing = 1.0
+        paragraphStyle.lineHeightMultiple = lineSpacing
+        paragraphStyle.alignment = self.textAlignment
+        
+        let attrString = NSMutableAttributedString(string: self.text!)
```

Comment:
This could actually live as its own var. Otherwise its unnecessary allocation of the attr string per call.